### ISSUE TITLE
LMSSE dots

### ISSUE THREAD

**Issue by ilia3101** [ilia3101/MLV-App#145]

**Comment by ilia3101** [comment]
>Found another implementation. The interfaces of this one are not very different to ours... just a little (if you see amaze interface - looks nearly identical. Edit: ah no... winx and winy are missing, so we would have to use OMP!).
https://github.com/Beep6581/RawTherapee/blob/dev/rtengine/demosaic_algos.cc
See function lmmse_interpolate_omp. It also has omp, so we could simplify our thread code - or we comment our omp and use our code... we would be free. It just has to be done. smile

>Edit: hm. Don't get it to work. What is LUTf ? And I don't get it to C - it will stay C++. disappointed_relieved

We don't need winy/winx or omp, just provide pointers to different lines of the image and give it smaller height value as input.

**Comment by masc4ii** [comment]
Yes... got that. No problem. But there are function calls in the code, and those functions are not there. LUTf for example. And for some C++ things I don't find the C equivalent... forgot what exactly. One could again convert interface to C->C++ and C++->C... if someone has fun doing so... 😞
